content/docs/Theory/Postprocessing/Thermodynamics.md

```diff
@@ -85,12 +85,12 @@ $$
 $$
 {{< /raw >}}
 
-### Grandpotetial in the GW approximation
+### Grand potential in the GW approximation
 
 
 
 [^Luttinger60]: J. M. Luttinger and J. C. Ward,  [Ground-State Energy of a Many-Fermion System. II](https://doi.org/10.1103/PhysRev.118.1417)
 [^Dahlen06]: Nils Erik Dahlen et al., [Variational energy functionals of the Green function and of the density tested on molecules](https://doi.org/10.1103/PhysRevA.73.012511)
 [^galitskii]: B. Holm and F. Aryasetiawan, [Total energy from the Galitskii-Migdal formula using realistic spectral functions](https://doi.org/10.1103/PhysRevB.62.4858)
 [^iskakov]: S. Iskakov, A. A. Rusakov, D. Zgid, and E. Gull, [Effect of propagator renormalization on the band gap of insulating solids](https://doi.org/10.1103/PhysRevB.100.085112)
-[^yeh]: C.-N. Yeh, S. Iskakov, D. Zgid, and E. Gull [Fully self-consistent finite-temperature 𝐺�𝑊 in Gaussian Bloch orbitals for solids](https://doi.org/10.1103/PhysRevB.106.235104)
\ No newline at end of file
+[^yeh]: C.-N. Yeh, S. Iskakov, D. Zgid, and E. Gull [Fully self-consistent finite-temperature ðºâð‘Š in Gaussian Bloch orbitals for solids](https://doi.org/10.1103/PhysRevB.106.235104)
```

Comment:
`[^yeh]: C.-N. Yeh, S. Iskakov, D. Zgid, and E. Gull [Fully self-consistent finite-temperature GW in Gaussian Bloch orbitals for solids](https://doi.org/10.1103/PhysRevB.106.235104)`
```suggestion
[^yeh]: C.-N. Yeh, S. Iskakov, D. Zgid, and E. Gull [Fully self-consistent finite-temperature GW in Gaussian Bloch orbitals for solids](https://doi.org/10.1103/PhysRevB.106.235104)
```